lib/process-services/task-list/components/attach-form.component.scss

```diff
@@ -0,0 +1,18 @@
+.adf-attach-form {
+    .mat-form-field {
+        width: 100%;
+    }
+
+    &-row {
+        display: flex;
+        justify-content: space-between;
+        margin: 20px 0;
+    }
+
+    .adf-no-form-mat-card-actions {
+        justify-content: flex-end;
+        margin-top: 30px;
+        text-align: right;
+        border-top: solid 1px #e0e0e0;
```

Comment:
why we have this custom and hardcoded colour? We should fetch the colour from the theme